appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/poa/campaign/CampaignMapper.java

```
@@ -53,5 +55,18 @@ private static String GetStringValue(String paramName, String response) {
 
     return "";
   }
+
+  public static Campaign mapCampaignFromBundle(Bundle response) {
+
+    try {
+      int responseCode = response.getInt("RESPONSE_CODE");
+      if (responseCode == ResponseCode.OK.getValue()) {
+        return new Campaign(new BigInteger(response.getString("CAMPAIGN_ID")), "");
+      }
+    } catch (Exception e) {
+      e.printStackTrace();
+    }
+    return new Campaign(new BigInteger("-1"), "");
```

Shouldn't we log an error here?
We might not have an exception and the campaign will be created with index -1 and fail silently